Examples/Pose3SLAMG2O/main.swift

```diff
@@ -45,10 +46,13 @@ struct FileHandlerOutputStream: TextOutputStream {
 
 func main() {
   // Parse commandline.
-  guard CommandLine.arguments.count == 3 else {
+  guard CommandLine.arguments.count > 3 else {
```

Comment:
Is there not a google flags like library that you can use?

Reply:
I hacked this through to avoid adding dependencies, can definitely improve later

Reply:
Nah, I think you should do it now :-)

Reply:
This is the most popular swift argument parsing library: https://github.com/apple/swift-argument-parser

Reply:
done

Reply:
@marcrasi Oh I didn't see your comment... I end up using SPMUtility but we can switch at anytime :)